tasks/main.yml

```diff
@@ -26,6 +26,16 @@
   register: autoclean_output
   changed_when: "'Del' in autoclean_output.stdout"
 
+- name: APT | Check for cached .deb files
+  shell: "ls /var/cache/apt/archives/*.deb 2&> /dev/null | wc -l"
+  register: apt_cache_check
+  changed_when: apt_cache_check.stdout != "0"
+  when: apt_clean
+
+- name: APT | Remove all cached .deb files
```

Comment:
The result of this command will always result in a `0` but it would definitely be cleaner with a single command.